server/src/test/scala/sbt/server/ProtocolTest.scala

```diff
@@ -621,13 +624,13 @@ class ProtocolTest {
           case ObjectToJson =>
             val json = addWhatWeWerePickling(t)(SerializedValue(t)(format.pickler))
             // IO.write(path, json.toString, IO.utf8)
-            if (path.exists) sys.error(s"$path exists already")
+            if (path.exists) () // Just ignore things we write twice, reading into different types should work.
```

Comment:
you could delete the line? but probably what it should really do is assert that what exists is identical to what we would have written, or something like that. I probably broke this with the Seq/List/Vector stuff, no? that was mostly just lazy... I could have made multiple files instead.